packages/headless/tsconfig.json

```diff
@@ -4,7 +4,7 @@
   "compilerOptions": {
     "rootDir": "./src",
     "baseUrl": ".",
-    "lib": ["DOM", "DOM.Iterable", "ES2022"],
+    "lib": ["DOM", "DOM.Iterable", "ES2024"],
```

Comment:
Why is this needed? Doesn't we force all using projects to add this to their tsconfig as well?